prometheus/base/alerting/prometheus.pod_alerts.yaml

```diff
@@ -25,3 +25,14 @@ spec:
       annotations:
         summary: "Pod {{ $labels.pod }} is crash looping"
         description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in waiting state (reason: 'CrashLoopBackOff') for more than 15 minutes."
+    - alert: PodsNotReady
+      expr: |
```

Comment:
I don't think the expression is correct.

As far as I understand it, all pods have entries for all phases all the time and the value is `1` for the entry with the current phase. I think we need to verify that the value is `== 1` rather than `>= 0`.

I think the current rule will alert constantly. You can verify this by adding entries for all phases on a unit test.

e.g.

```
      - series: 'kube_pod_status_phase{pod="pod-1", namespace="ns-1", phase="Pending"}'
        values: '0x15'
      - series: 'kube_pod_status_phase{pod="pod-1", namespace="ns-1", phase="Unknown"}'
        values: '0x15'
      - series: 'kube_pod_status_phase{pod="pod-1", namespace="ns-1", phase="Failed"}'
        values: '0x15'
      - series: 'kube_pod_status_phase{pod="pod-1", namespace="ns-1", phase="Running"}'
        values: '1x15'
      - series: 'kube_pod_status_phase{pod="pod-1", namespace="ns-1", phase="Succeeded"}'
        values: '0x15'
      - series: 'kube_pod_status_unschedulable{namespace="ns-1", pod="pod-1"}'
        values: '0x15'
```

Reply:
I've updated the test case file according to the new expression.